src/main/java/com/metamx/emitter/core/HttpPostEmitter.java

```diff
@@ -303,6 +303,14 @@ public void run()
 
             final StatusResponseHolder response = client.go(request, new StatusResponseHandler(Charsets.UTF_8)).get();
 
+            if (response.getStatus().getCode() == 413) {
+              throw new ISE(
+                  "Received HTTP status 413 from %s. Batch size of %,d may be too large, try adjusting com.metamx.emitter.http.maxBatchSizeBatch",
```

Comment:
can we add brackets around the values, just to be consistent with other log messages?
Also, we don't typically add thousands separators to numbers, unless I'm mistaken.